components/composed/model/ModelList/hooks/useModelList.tsx

```diff
@@ -68,7 +68,7 @@ function useModelList<
     () => ({
       ...(actions.handleEdit && { edit: { handleClick: actions.handleEdit } }),
       ...(actions.handleDelete && {
-        delete: { handleClick: actions.handleDelete },
+        delete: { handleClick: actions.handleDelete, modalConfirm: true },
```

Comment:
Adding the flag to actions that require a modal. Could also be passed in from the List components themselves.